CONTRIBUTING.md

```diff
@@ -12,16 +12,30 @@ file.
 
 ## Development prep
 
-- `git clone https://github.com/flutter/devtools`
-- `cd devtools/packages/devtools_app`
-- `flutter pub upgrade`
+1. If you haven't already, follow the [instructions](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh) to generate a new SSH key and connect to Github with SSH
+2. Follow the [instructions](https://docs.github.com/en/get-started/quickstart/fork-a-repo) to fork the DevTools repo to your own Github account, and clone using SSH
+3. Make sure to [configure Git to keep your fork in sync](https://docs.github.com/en/get-started/quickstart/fork-a-repo#configuring-git-to-sync-your-fork-with-the-original-repository) with the main DevTools repo
+4. Finally, update to the newest version of the package with `cd packages/devtools_app` and `flutter pub upgrade`
 
 From a separate terminal, start running a flutter app to connect to DevTools:
 - `git clone https://github.com/flutter/gallery.git` (this is an existing application with many examples of Flutter widgets)
 - `cd gallery`
 - ensure the iOS Simulator is open (or a physical device is connected)
 - `flutter run`
 
+## Workflow for making changes
+
+- Create a branch from your cloned repo: `git checkout -b myBranch`
+- Commit work to your branch: `git commit -m “description”`
+- Push to your branch: `git push origin myBranch`
+- Navigate to the Pull Requests tab in the main [DevTools repo](https://github.com/flutter/devtools). You should see a popup to create a pull request from your local branch to DevTools master. Create a pull request.
+- Ping your PR link to whomever you want to review it
```

Comment:
I think you can delete this step - this is more internal common practice. An non-google user reading these instructions wouldn't have access to "ping" a reviewer so we may want to leave this out.